docs/index.md

```diff
@@ -11,15 +11,40 @@ This is the documentation for the `cfgov-refresh` project, a redesign of the [ww
 The standard technology stack for development of cfgov-refresh within the CFPB consists of the following base:
 
 - macOS
+- [Apache]() - Version? Which project exactly? https://apache.org/index.html#projects-list
 - [Homebrew](https://brew.sh) - package manager for installing system software on OSX
-- Python and PIP (Python package manager)
-- [Jinja templates](http://jinja.pocoo.org) for front-end rendering
-- [Wagtail CMS](https://wagtail.io) for content administration
-- Dependencies, listed below
+- Python 2.7 and PIP (Python package manager)
+- [Jinja2 templates](http://jinja.pocoo.org/docs/2.10/) for front-end rendering. See [`requirements/libraries.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/libraries.txt) for version.
+- [Wagtail CMS](https://wagtail.io) for content administration. See [`requirements/wagtail.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/wagtail.txt) for version.
+- [PostgreSQL](https://www.postgresql.org/) and [Psycopg](http://initd.org/psycopg/) for our database. See [`requirements/postgres.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/postgres.txt) for version.
+- Additional dependencies, listed below
```

Comment:
What is the philosophical difference between these two lists?

Reply:
Runtime dependencies vs build dependencies?

Reply:
I suppose that makes sense, but in that case, Elasticsearch should move up.